widgets/src/components/ProcessForm/ProcessFormContainer.js

```diff
@@ -78,23 +80,52 @@ class ProcessFormContainer extends React.Component {
     return null;
   }
 
+  submitProcessForm(form) {
+    this.setState({ submitting: true }, async () => {
+      const { config } = this.state;
+      const { onSubmitForm } = this.props;
+
+      const connection = (config && config.knowledgeSource) || '';
+      const processContainerId = (config && config.process) || '';
+
+      try {
+        const response = await postProcessForm(connection, processContainerId, form.formData);
+        onSubmitForm({ ...form, response });
+      } catch (error) {
+        this.handleError(error.message);
```

Comment:
Apologies for not mentioned this before, we have updated the error handling a bit as onError via custom events is not implemented yet, could you update it to support local error notifications as well? You can refer to https://github.com/entando/entando-process-driven-plugin/blob/develop/widgets/src/components/TaskComments/TaskCommentsContainer.js#L189 for implementation details - basically handleError additionally sets state's errorMessage, error notification component is added at top level, and close notification function just cleans the errorMessage.